eng/AfterSolutionBuild.targets

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
 
-  <Target Name="_PublishLanguageServerRids" AfterTargets="Pack" Condition="'$(DotNetBuildFromSource)' != 'true'">
+  <Target Name="_PublishLanguageServerRids" AfterTargets="Pack" Condition="'$(DotNetBuildFromSourceOnly)' != 'true' and '$(ArcadeBuildFromSourceOnly)' != 'true'">
```

Comment:
This looks incorrect. razor is on 8.0 arcade so this needs to be compatible with 8.0 and 9.0 arcade properties. `ArcadeBuildFromSourceOnly` isn't actually a switch. I think you want:

```suggestion
  <Target Name="_PublishLanguageServerRids" AfterTargets="Pack" Condition="'$(DotNetBuildFromSourceOnly)' != 'true' and '$(DotNetBuildFromSource)' != 'true'">
```